src/requirements/data/majors/fsad.ts

```diff
@@ -0,0 +1,219 @@
+import { Course, CollegeOrMajorRequirement } from '../../types';
+import { includesWithSubRequirements, ifCodeMatch, courseIsForeignLang } from '../checkers-common';
+
+const fashionDesignAdditionalRequirements: readonly string[] = [
+  'PBS',
+  'BIOLS-AG',
+  'BIONLS-AG',
+  'SBA',
+  'KCM',
+  'MQR',
+  'LA',
+  'CA',
+  'HA',
+];
+
+const fashionDesignNaturalScienceRequirements: readonly string[] = ['PBS', 'BIOLS-AG', 'BIONLS-AG'];
+
+const fashionDesignRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Social Science',
```

Comment:
let's add "disallowTransferCredit: true" here as a temp fix for now bc AP stats is counting for this but it shouldn't

here's of an example of when we use it:
<img width="883" alt="Screen Shot 2022-04-01 at 1 55 07 AM" src="https://user-images.githubusercontent.com/14082828/161203575-08525193-cbe0-433f-a833-c81a306048c5.png">